sale_order_note_template/models/sale_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    terms_template_id = fields.Many2one(
+        "sale.terms_template",
+        string="Terms and conditions template",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    note = fields.Html(readonly=True, states={"draft": [("readonly", False)]})
```

Comment:
If I understand correctly. you change the type of the field ``note``.
you should maybe add a ``post_init_hook`` and / or a ``uninstall_hook`` to handle correctly the conversion (``Text`` <--> ``Html``). Otherwise, I guess if you uninstall the module, you'll have a lot of html anchors in all the ``note`` column. Don't you think ? (or maybe there is a core feature in Odoo to handle such conversion, I must confess that I have not tested to install your module, add some text in the ``note`` field, and then, uninstall the module.)

If it's not in the scope of your PR, you can make a mention in a ROADMAP.rst section.

Reply:
I'll move that in roadmap at the moment !

thanks for review